ApiReview/API.Polly.Core/NoDocs/Polly/ResiliencePipeline.T.cs

```diff
@@ -0,0 +1,25 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Polly.Utils;
+
+namespace Polly;
+
+public sealed class ResiliencePipeline<T>
+{
+    public static readonly ResiliencePipeline<T> Null;
```

Comment:
does the word Null reflect the purpose of the pipeline? would something like Empty or PassThrough be better?

Reply:
I think `Empty` is a better name. Originally, this was modeled after `NullLoggerFactory` but now we don't have to use the null token.

Reply:
Addressed in #1506